apps/transport/lib/transport_web/controllers/contact_controller.ex

```diff
@@ -36,7 +36,74 @@ defmodule TransportWeb.ContactController do
   end
 
   def send_mail(conn, params) do
-    Logger.error("Bad parameters for sending email #{params}")
+    Logger.error("Bad parameters for sending email #{inspect(params)}")
+
+    conn
+    |> put_flash(:error, gettext("There has been an error, try again later"))
+    |> redirect(to: params["redirect_path"] || page_path(conn, :index))
+  end
+
+  def send_feedback(conn, %{"feedback" => %{"name" => name, "email" => email}} = params) when name !== "" do
+    # someone filled the honeypot field ("name") => discard as spam
+    Logger.info("Feedback coming from #{email} has been discarded because it filled the feedback form honeypot")
+
+    conn
+    # spammer get a little fox emoji in their flash message, useful for testing purpose
+    |> put_flash(:info, "Your feedback has been sent, we will contact you soon 🦊")
+    |> redirect(to: params["redirect_path"] || page_path(conn, :index))
+  end
+
+  def send_feedback(
+        conn,
+        %{"feedback" => %{"rating" => rating, "explanation" => explanation, "email" => email, "feature" => feature}}
+      )
+      when rating in ["like", "neutral", "dislike"] and
+             feature in ["gtfs-stops", "on-demand-validation", "gbfs-validation"] do
+    [email, explanation] = [email, explanation] |> Enum.map(&String.trim/1)
+
+    rating_t = %{"like" => "j’aime", "neutral" => "neutre", "dislike" => "mécontent"}
+
+    feedback_content = """
+    Vous avez un nouvel avis sur le PAN.
+    Fonctionnalité: #{feature}
+    Notation: #{rating_t[rating]}
+    Adresse email: #{email}
+
+    Explication: #{explanation}
+    """
+
+    reply_email =
+      if email == "" do
+        Application.fetch_env!(:transport, :contact_email)
+      else
+        email
+      end
+
+    case Transport.EmailSender.impl().send_mail(
+           "PAN, Formulaire Feedback",
+           Application.fetch_env!(:transport, :contact_email),
+           Application.fetch_env!(:transport, :contact_email),
+           reply_email,
+           "Nouvel avis pour #{feature}: #{rating_t[rating]}",
```

Comment:
```suggestion
           "Nouvel avis pour #{feature} : #{rating_t[rating]}",
```

Adapter les tests si tu matches dessus